Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG GO_VERSION=1.15
+ARG FROM_IMAGE=alpine:latest
+
+FROM golang:${GO_VERSION}-alpine3.12 AS builder
+RUN apk add --update --no-cache bash ca-certificates make curl git mercurial tzdata
+
+# Install kubectl
+ARG KUBECTL_VERSION=v1.16.1
+RUN curl -L -s https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/amd64/kubectl -o /usr/local/bin/kubectl
+RUN curl -L -s https://amazon-eks.s3-us-west-2.amazonaws.com/1.14.6/2019-08-22/bin/linux/amd64/aws-iam-authenticator -o /usr/local/bin/aws-iam-authenticator
```

Comment:
**Question**: shouldn't we put the AWS IAM authenticator version (`1.14.6`) into an argument as well?

Reply:
Unfortunately, the date is contained by the URL, so the proper date should be passed as well.

Reply:
Indeed.